### PR TITLE
Removed equiv before performing unit conversion

### PR DESCRIPTION
**Pull request by Rlamboll**
# Please confirm that this PR has done the following:

- [x] Tests Added
- [x] Documentation Added
- [x] Description in RELEASE_NOTES.md Added

## Adding to RELEASE_NOTES.md (remove section after adding to RELEASE_NOTES.md)

Please add a single line in the release notes similar to the following:

```
- (#XX)[http://link-to-pr.com] Added feature which does something
```

# Description of PR

Please describe the changes introduced by this PR.

### CODE REVIEW

**Comment by Rlamboll** [comment]
Seems to have failed a test for time reasons, I don't think I can restart it though

**Comment by danielhuppmann** [comment]
Yes, there is some version-conflict in the py3.6 dependencies - guess we'll need to remove that test at some point...

File: pyam/units.py [hunk 1]
```diff
@@ -26,21 +26,24 @@ def convert_unit(df, current, to, factor=None, registry=None, context=None,
     # Convert using a pint.UnitRegistry; default the one from iam_units
     registry = registry or iam_units.registry
 
+    # Make versions without -equiv
```
Comment (danielhuppmann) [inline pyam/units.py:29]: I think that having an own function for this simple removal is a bit excessive, you could do that in one line with 
```
current, _to = [i.replace('-equiv', 'e') for i in [current, to]]
```
(note the `_to` as an alternative to kee p `to` for the renaming)

File: tests/test_units.py [hunk 2]
```diff
@@ -46,6 +47,22 @@ def test_convert_unit_with_pint(test_df, current, to):
     assert_converted_units(df, current, to, exp)
 
 
+def test_convert_eqiv_units(test_df):
```
Comment (danielhuppmann) [inline tests/test_units.py:50]: Should be enough to just add a line to the parametrisation of `test_convert_gwp()`?

Comment (Rlamboll) [reply tests/test_units.py:50]: Can do, but I don't really trust this as about a quarter of these tests are broken on my machine, I don't understand why.

File: pyam/units.py (hunk 1 above)
Comment (Rlamboll) [reply pyam/units.py:29]: By leaving out the 'e' this works for all units, even ones without the alias set up.

File: tests/test_units.py (hunk 2 above)
Comment (danielhuppmann) [reply tests/test_units.py:50]: did you install the latest version of `iam_units` from GitHub?

Comment (Rlamboll) [reply tests/test_units.py]: nope, turns out it had moved on

**Comment by danielhuppmann** [comment]
thanks @Rlamboll, looks great - can you still pull my suggested changes in https://github.com/Rlamboll/pyam/pull/2 (or do you disagree?), then good to go!

**Comment by Rlamboll** [comment]
Honestly I find those variable names very confusing because unless you read that specific line of code, you'd have no idea what those variables were and I would likely assume current was still the input value if I wanted to use it later.

**Comment by danielhuppmann** [comment]
> Honestly I find those variable names very confusing because unless you read that specific line of code, you'd have no idea what those variables were and I would likely assume current was still the input value if I wanted to use it later.

I pushed another commit to the PR-branch so that the "cleaned" (without `-equiv`) unit-args are both saved as new internal variables. But I think that cleaning `current` to `_current` is much more python than `clean_current`...

Still, if you can't be convinced, please cherry-pick  the other commits (removed unused utility function and add extra test).

**Comment by Rlamboll** [comment]
Well, you're the one doing the upkeep so if you want those names you can have them.

**Pull request merged**